db/schema.rb

```diff
@@ -9,154 +9,153 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended to check this file into your version control system.
+# It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20190420025012) do
+ActiveRecord::Schema.define(version: 20190420025012) do
 
-  create_table "annotations", :force => true do |t|
-    t.integer  "map_id"
-    t.integer  "user_id"
-    t.string   "annotation_type"
-    t.string   "text"
-    t.string   "style"
-    t.string   "coordinates"
-    t.datetime "created_at",      :null => false
-    t.datetime "updated_at",      :null => false
+  create_table "annotations", force: :cascade do |t|
+    t.integer  "map_id",          limit: 4
+    t.integer  "user_id",         limit: 4
+    t.string   "annotation_type", limit: 255
+    t.string   "text",            limit: 255
+    t.string   "style",           limit: 255
+    t.string   "coordinates",     limit: 255
+    t.datetime "created_at",                  null: false
+    t.datetime "updated_at",                  null: false
   end
```

Comment:
How did these schema changes happen without any migrations?

Reply:
Was it done by rubocop?

Reply:
Yes, rubocop has formatted it. But there was no impact on database setup or anything like that.